pt-PT/Universally Preferable Behaviour/p02_ch03_the_second_test_murder.md

```diff
@@ -1,39 +1,39 @@
-## The Second Test: Murder
+## Segundo Teste: Assassinato
 
-Let us now test the UPB framework against moral propositions regarding *murder*, which here is defined as killing intentionally and with premeditation, not in self-defence.
+Deixe-nos testar a estrutura do UPB contra proposições morais sobre o *assassinato*, que aqui é definido como matar intencionalmente e com premeditação, não em legítima defesa.
 
-Since we spent so much time dissecting the question of rape – and since many of the same arguments will apply here – this analysis can be much briefer.
+Uma vez que passamos tanto tempo dissertando a questão da violação – e já que vários dos mesmos argumentos vão ser aplicados aqui – esta análise pode ser muito mais breve.
 
-Let us return to our two moral guinea pigs sitting in a room – we’ll call them Bob and Doug.
+Voltemos para as nossas duas cobaias morais sentados num quarto - iremos chama-los de Bob e Doug.
 
-If murder is morally good, then clearly *refraining* from murder is immoral. Thus the only chance that Bob and Doug have to be moral is in the instant that they simultaneously murder each other. Physically, this is impossible of course – if they both stand and grip each other’s throats, they will never succumb to strangulation at exactly the same moment. If Bob dies first, his grip on Doug’s throat will loosen, thus condemning Doug to the status of *immorality* until such time as he can find another victim. Because Bob dies first – and thus cannot continue to try murdering Doug – Bob’s death renders him more immoral than Doug’s murder.
+Se o assassinato é moralmente bom, então claramente *abster-se* de assassinar é imoral. Assim, a única hipótese que o Bob e o Doug têm de ser moral é no instante em que eles simultaneamente se assassinarem um ao outro. Fisicamente, isto é impossível como é obvio – se os dois estão de pé e ambos agarrarem o pescoço um do outro, eles nunca vão sucumbir por estrangulamento ao mesmo tempo. Se o Bob morrer primeiro, o aperto na garganta do Doug será perdido, assim condenando Doug ao status de *imoralidade*, até ao momento que ele encontre outra vítima. Como o Bob morre primeiro - e assim não pode continuar a tentar matar o Doug - a morte do Bob rende-lhe mais imoral do que o assassinato de Doug.
```

Comment:
The phrase is quite obscure. 
"a única hipótese que o Bob e o Doug têm de ser moral é" looks like an automated translation to me.
For example, "a única forma do Bob e Doub serem morais é quando, no mesmo instante, eles  assassinarem-se simultaneamente".

Reply:
To be honest, I had a hard time translating this sentence, because I could not find the meaning of it. Your solution is much better.